Cirrious.FluentLayout/FluentLayoutExtensions.cs

```diff
@@ -18,8 +18,8 @@ public static void SubviewsDoNotTranslateAutoresizingMaskIntoConstraints(this UI
             foreach (var subview in view.Subviews)
             {
                 subview.TranslatesAutoresizingMaskIntoConstraints = false;
-		
-	        if (subview.Subviews != null && subview.Subviews.Count() > 0)
+
+                if ((subview as UIControl) == null && subview.Subviews != null && subview.Subviews.Count() > 0)
```

Comment:
@Cheesebaron   oh, yes, you are right.  So ignore the changes, any suggestion for the subviews Constraints issue at here?